recipes/yaff/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "yaff" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7bd303f8a2c34d139cbba830b18ebfadb890d8a45f48e160c58dc66c974bba78
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [not linux or py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy>=1.5
```

Comment:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy on how to correctly add numpy dependency

Reply:
Can you please do the same for the other recipe in this PR?

Reply:
I did it in https://github.com/conda-forge/staged-recipes/pull/8894